tests/net/mod.rs

```diff
@@ -502,7 +502,7 @@ where
 
         let trace = self.trace.unwrap_or_else(|| {
             // If the trace setting is not overriden, we use the setting from the environment.
-            let setting = env::var("HBBFT_TEST_TRACE").unwrap_or_else(|_| "true".to_string());
+            let setting = env::var("HBBFT_TEST_TRACE").unwrap_or_else(|_| "false".to_string());
```

Comment:
We could avoid creating another string:

```rust
env::var("HBBFT_TEST_TRACE").map_or_else(false, |setting| setting == "true" || setting == "1")
```

Reply:
We could, I considered it and chalked it up to copying it verbatim from elsewhere. Shame on me!